etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -0,0 +1,36 @@
+definitions:
+  common: |
+    This study estimates the annual number of farmed fish slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available fish size data (from a range of internet sources).
+
+    Note that this estimate does not include:
+    - Non fish species, e.g. lobsters.
+    - Fish deaths arising in fish farms prior to harvest, in the capture of wild fish for feed or re-stocking, or from environmental impacts on surrounding wildlife.
+    - Fish farmed for use as bait, whose small production tonnages may comprise large numbers of fish due to their very small size of just a few grams.
+    - Species without estimated mean weight (which could be an additional 17% of total fish production).
+tables:
+  number_of_farmed_fish:
+    variables:
+      n_farmed_fish_low:
+        title: Estimated number of farmed fish (lower bound)
+        unit: 'fish'
+        short_unit: ''
+        description: |
+          Lower bound estimate of the number of farmed fish.
+
+          {definitions.common}
+      n_farmed_fish:
+        title: Estimated number of farmed fish
+        unit: 'fish'
+        short_unit: ''
+        description: |
+          Midpoint estimate of the number of farmed fish in the world, calculated as the average between the lower and upper limit estimates.
+
+          {definitions.common}
+      n_farmed_fish_high:
+        title: Estimated number of farmed fish (upper bound)
+        unit: 'fish'
+        short_unit: ''
+        description: |
+          Upper bound estimate of the number of farmed fish.
+
+          {definitions.common}
```

Comment:
Remove the line below this, as it gets reflected on the metadata

Reply:
Hmm, I've just noticed that the three indicators have an additional line at the end, not just this one. I don't know how to remove it, but it doesn't bother me.

Reply:
Ah, ok, this is fixed by using `|-` in the common definition, as you suggest below.